core/src/main/web/app/mainapp/components/notebook/cell-directive.js

```diff
@@ -179,17 +179,20 @@
 
         $scope.cellview.menu.addItem({
           name: 'Delete cell',
+          shortcut: bkUtils.isWindows ? bkUtils.trustAsHtml('Ctrl-Alt-D') : bkUtils.trustAsHtml('&#x2318;-Alt-D'),
```

Comment:
there is a lot of repeated code here.
1) instead of putting the conditional on windows into every shortcut, the shortcut should have a form with two values for the two platforms (Linux and Windows keybindings are the same).  so how about:
```
shortcut: ['Ctrl-Alt-D', 'Cmd-Alt-D']
```
Then on Mac, Cmd gets replaced with &#x2318; and Alt with &#x2325;